src/mpi/init/netloc_util.c

```diff
@@ -36,13 +40,14 @@ static int get_tree_attributes(netloc_topology_t topology,
         (netloc_dt_lookup_table_t *) MPL_malloc(sizeof(netloc_dt_lookup_table_t), MPL_MEM_OTHER);
     mpi_errno = netloc_get_all_host_nodes(topology, host_nodes);
 
+    MPIR_CHKPMEM_DECL(3);
```

Comment:
Declarations should go to the top of the function.

Reply:
will fix